### ISSUE TITLE
Compatability with hid-remapper?

### ISSUE THREAD

**Issue by 222Phoenix** [hrvach/deskhop#154]
Is there a way to get deskhop to work with this?

https://github.com/jfedor2/hid-remapper/tree/master

**Comment by hrvach** [comment]
Did you try it? I think there is a chance it might work, but I've never tried. Looks like a very cool project!

**Comment by 222Phoenix** [comment]
Yes, it doesn’t work. It’s hard to explain because it’s hard to tell what’s going on. Cursor movement rapidly presses all kinds of random keyboard keys.

**Comment by 222Phoenix** [comment]
Update, it's working with Desktop 0.71 at the moment. It may depend on one or both firmwares.

**Comment by hrvach** [comment]
> Update, it's working with Desktop 0.70 at the moment. It may depend on one or both firmwares.

That's great to hear! Some bugs were indeed fixed for 0.7...
